libr/core/core.c

```diff
@@ -194,6 +194,8 @@ R_API int r_core_bind(RCore *core, RCoreBind *bnd) {
 	bnd->cmd = (RCoreCmd)r_core_cmd0;
 	bnd->cmdstr = (RCoreCmdStr)r_core_cmd_str;
 	bnd->puts = (RCorePuts)r_cons_strcat;
+	bnd->num = core->num;
```

Comment:
i do need core->num; this is the value that gets used for testing the result of the macro execution. i have tried using dbg->num in debug.c, but it does not get updated with proper value. Is there any other way to access the return value of a command executed by r_core_cmd0 without access to core->num?